nbodykit/cosmology/power/galaxy.py

```diff
@@ -0,0 +1,128 @@
+import numpy
+from . import transfers
+from ..cosmology import Cosmology
+from .linear import LinearPower
+
+class GalaxyPower(object):
+    """
+    An object to compute the galaxy/redshift power spectrum and related quantities,
+    using a transfer function from the CLASS code or the analytic
+    Eisenstein & Hu approximation.
+
+    Parameters
+    ----------
+    cosmo : :class:`Cosmology`, astropy.cosmology.FLRW
+        the cosmology instance; astropy cosmology objects are automatically
+        converted to the proper type
+    redshift : float
+        the redshift of the power spectrum
+    transfer : str, optional
+        string specifying the transfer function to use; one of
+        'CLASS', 'EisensteinHu', 'NoWiggleEisensteinHu'
+    b0 : float
+        the linear bias of the galaxy in a gaussian field
+    fnl : float
+        the non-gaussian parameter
```

Comment:
"Primordial non-gaussian parameter"?

Reply:
Thanks. I have made all those changes and updated the repo. I will add the unit test part soon. Thanks again.